frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/InnerContainerBox.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, styled } from '@mui/material';
+
+export const StyledInnerBox = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    backgroundColor: theme.palette.background.default,
+    border: `1px solid ${theme.palette.divider}`,
+    padding: theme.spacing(2),
+    borderRadius: `${theme.shape.borderRadiusMedium}px`,
+}));
+
+export const InnerBoxHeader = styled('div')(({ theme }) => ({
+    marginLeft: 'auto',
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+        display: 'none',
+    },
+}));
+
+// row for inner containers
+export const Row = styled('div')({
+    display: 'flex',
+    flexDirection: 'row',
+    width: '100%',
+});
+
+export const Col = styled('div')({
+    flex: 1,
+    margin: '0 4px',
+});
+
+export const BoxSeparator = ({ text }: { text: string }) => {
```

Comment:
Consider using { children } instead, the default slot in React. That way you can use your component as `<BoxSeparator>My text...</BoxSeparator>`